packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -77,68 +119,27 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
     }
   }, [progress, showProgressIndicator, attachmentName, uploadStartedString, uploadCompletedString]);
 
-  const progressBarThicknessPx = 4;
-
   const containerClassName = mergeStyles({
     width: '12rem',
-    background: theme.palette.neutralLighter,
-    borderRadius: theme.effects.roundedCorner4,
-    border: `${_pxToRem(1)} solid ${theme.palette.neutralQuaternary}`,
-    cursor: 'pointer'
+    minWidth: '75%'
```

Comment:
How do width and minWidth interact with each other?

Reply:
The width declaration will be set 12rem. The min-width makes it so that element has to be at least 75% of the parent wide.